CLAUDE.md

```
@@ -0,0 +1,74 @@
+
+## 言語
+
+日本語で回答すること。
+
+## Overview
+
+AWS infrastructure managed with OpenTofu (Terraform 互換)。Region: ap-northeast-1。Multi-account structure using AWS Organizations with IAM Identity Center (SSO) for authentication。
+
+### ディレクトリ構成
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)